Thu Mar  5 12:22:30 IST 2020 crash report /var/crash/_usr_share_apport_apport-gtk.1000.crash detected
/usr/share/apport/apport-gtk:16: PyGIWarning: Wnck was imported without specifying a version first. Use gi.require_version('Wnck', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import GLib, Wnck, GdkX11, Gdk
Traceback (most recent call last):
  File "/usr/share/apport/apport-gtk", line 594, in <module>
    app.run_argv()
  File "/usr/lib/python3/dist-packages/apport/ui.py", line 682, in run_argv
    return self.run_crashes()
  File "/usr/lib/python3/dist-packages/apport/ui.py", line 233, in run_crashes
    logind_session[1] > self.report.get_timestamp():
TypeError: unorderable types: float() > NoneType()
